README.md

```diff
@@ -34,17 +34,17 @@ Here are two prompts, each for executing different analysis processes.
 ```
 1st Prompt: Initialize the rule instance with the rule path set to "rule.json"
 ```
-Used Quark Script APIs/Tools that LLM used:  `Rule()`
+Used Quark Script APIs/Tools that LLM used:  `Rule`
 
 ```
 2nd Prompt: Run Quark Analysis using the rule instance on the apk sample "ovaa.apk", 
             and Check if the parameters are hard-coded. If yes, display the hard-coded values.
 ```
-Used Quark Script APIs/Tools that LLM used: `runQuarkAnalysis()`, `getParameterValues()` and `isHardCoded()`
+Used Quark Script APIs/Tools that LLM used: `runQuarkAnalysis`, `behaviorOccurList`, `getParameterValues` and `isHardCoded`
```

Comment:
In quarkScriptAgent.py,  the tool `behaviorOccurList` is named `getBehaviorOccurList`. I suggest keeping the name consistent.